myning/chapters/enter_garden.py

```diff
@@ -40,7 +39,7 @@ def get_seeds():
                 seeds.remove(seed)
                 player.inventory.add_item(seed)
                 FileManager.save(seed)
-            FileManager.save(player)
+            FileManager.multi_save(player, garden)
```

Comment:
This will look less annoying when we pull inventory out. It should be the next itme.